app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      before_action :authenticate_user!, except: [:create]
```

Comment:
instead of doing a before_action and add the except, you could use `skip_before_action`, docs [here](https://guides.rubyonrails.org/action_controller_overview.html#filters)